sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -200,17 +200,19 @@ def from_connection_string(
             conn_str, credential, 'queue')
         if 'secondary_hostname' not in kwargs:
             kwargs['secondary_hostname'] = secondary
-        return cls(account_url, queue_name=queue_name, credential=credential, **kwargs) # type: ignore
+        return cls(account_url, queue_name=queue_name, credential=credential, **kwargs)
 
     @distributed_trace
-    def create_queue(self, **kwargs):
-        # type: (Any) -> None
+    def create_queue(
+            self, metadata: Optional[Dict[str, str]] = None,
```

Comment:
You'll need to add a `*,` line since `metadata` is a keyword. You've made it an ordered param as is.

```py
def create_queue(
    self, *,
    metadata: Optional[Dict[str, str]] = None,
    ...
```